app/js/cohortbuilder.jsx

```diff
@@ -10,22 +10,15 @@
 
 import React from 'react';
 import {render} from 'react-dom';
-
 import {Router, Route, hashHistory} from 'react-router';
-import {Provider} from 'react-redux';
+import App from './components/App';
 
-import createStore from './redux-store';
-import routes from './routes';
 // css necessary for custom react-select input fields which would be used 
```

Comment:
are this comments necessary? 
Otherwise good job

Reply:
Thank you for pointing that out.  Resolved.